src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-password-form/user-password-form.component.html

```diff
@@ -0,0 +1,117 @@
+<div class="col-sm-12 col-lg-6">
+  <form class="form-horizontal"
+        #frm="ngForm"
+        [formGroup]="userForm"
+        novalidate>
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h3 i18n="form title|Example: Create Pool@@formTitle"
+            class="panel-title">{{ action | titlecase }} {{ resource | upperFirst }}</h3>
+      </div>
+      <div class="panel-body">
+
+        <!-- Old password -->
+        <div class="form-group"
+             [ngClass]="{'has-error': userForm.showError('oldpassword', frm)}">
+          <label class="control-label col-sm-3"
+                 for="oldpassword">
+            <ng-container i18n>Old password</ng-container>
+            <span class="required"></span>
+          </label>
+          <div class="col-sm-9">
+            <div class="input-group">
+              <input class="form-control"
+                     type="password"
+                     placeholder="Old password..."
+                     id="oldpassword"
+                     formControlName="oldpassword"
+                     autocomplete="off"
+                     autofocus>
+              <span class="input-group-btn">
+                <button type="button"
+                        class="btn btn-default"
```

Comment:
`btn-default` has been dropped.
We now have `btn-primary` (blue) `btn-secondary` (red) and `btn-light` ( white )
```suggestion
                        class="btn btn-light"
```